### PR TITLE
sync check productionalize

### PR DESCRIPTION
**Pull request by LesnyRumcajs**
Made sync check great again. Please refer to https://github.com/ChainSafe/forest-iac/pull/7/files#diff-d6dcb1e59b16334606053e995822efabbf804c9b98b94f606f830cf0ee9ef3a0.

As a bonus, there should be close to no maintenance (unless unforeseen fiascoes arise) thanks to automatic updating and cleaning.

Closes https://github.com/ChainSafe/forest/issues/1709

### CODE REVIEW

**Comment by LesnyRumcajs** [comment]
Moved to draft, something fiascoed in Ruby Slack API, I'll investigate it first.

**Comment by LesnyRumcajs** [comment]
There was a fiasco with me tinkering with my test slack channel, nothing PR-related.

File: scripts/sync_check/sync_check_process.rb [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+  # Cleans up the sync check
+  def cleanup
+    @logger.info 'Cleaning up sync check'
+    @client.post_message '🧹 Cleaning up sync check'
+
+    FileUtils.rm_rf(Dir.glob("#{FOREST_DATA}/**"))
+
+    @client.attach_comment '🧹 Cleanup finished ✅'
+  end
```
Comment (lemmih) [inline scripts/sync_check/sync_check_process.rb:66]: Should the services be stopped before clean-up?

**Review by lemmih — APPROVED** [review]
Lovely.

**Pull request merged**